backend/python_init_script/main.py

```diff
@@ -2,41 +2,186 @@
+        roles=user_data.roles,
+        url=os.getenv("APPWRITE_ENDPOINT"),
+        name=user_data.name,
+    )
+
+
+def add_user(users: Users, user_data: UserData):
+    """
+    Creates user account, according to the user_data
+    """
+    users.create(email=user_data.email, password=user_data.init_pw, name=user_data.name)
+
+
+if __name__ == "__main__":
+    import argparse
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument("file", type=str, help="Path of file containing the user data")
+    parser.add_argument(
+        "--apikey",
+        type=str,
+        default=None,
+        help="API key to access appwrite backend, you can instead "
+        "also set the environment variable APPWRITE_API_KEY",
+    )
+    parser.add_argument(
+        "--endpoint",
+        type=str,
+        default=None,
+        help="URL of the appwrite endpoint, you can instead "
+        "also set the environment variable APPWRITE_ENDPOINT",
+    )
+    parser.add_argument(
+        "--projectid",
+        type=str,
+        default=None,
+        help="Project ID of the targeted appwrite project, you can instead "
+        "also set the environment variable APPWRITE_PROJECT_ID",
+    )
+    args = parser.parse_args()
+
+    _key = str(args.apikey) if args.apikey else os.getenv("APPWRITE_API_KEY")
+    _end = str(args.endpoint) if args.endpoint else os.getenv("APPWRITE_ENDPOINT")
+    _pid = str(args.projectid) if args.projectid else os.getenv("APPWRITE_PROJECT_ID")
 
-result = teams.create("Admins")
-_id = result["$id"]
-_endpoint = os.environ.get("APPWRITE_ENDPOINT")
+    if None in [_key, _end, _pid]:
+        print(
+            "Appwrite API key, project id and endpoint url must be provided "
+            "via command line arguments or environment variables"
+        )
+        exit()
 
-# fmt: off
-teams.create_membership(team_id=_id, email="admin0@admin1.de", roles=['owner'], url=_endpoint, name="Berinike")
-teams.create_membership(team_id=_id, email="admin1@admin1.de", roles=['owner'], url=_endpoint, name="Dominic")
-teams.create_membership(team_id=_id, email="admin2@admin1.de", roles=['owner'], url=_endpoint, name="Felix")
-teams.create_membership(team_id=_id, email="admin3@admin1.de", roles=['owner'], url=_endpoint, name="Nick S.")
-teams.create_membership(team_id=_id, email="admin4@admin1.de", roles=['owner'], url=_endpoint, name="Christoph")
-teams.create_membership(team_id=_id, email="admin5@admin1.de", roles=['owner'], url=_endpoint, name="Ba Que")
-teams.create_membership(team_id=_id, email="admin6@admin1.de", roles=['owner'], url=_endpoint, name="Nik N.")
-teams.create_membership(team_id=_id, email="admin7@admin1.de", roles=['owner'], url=_endpoint, name="Jannis")
-# fmt: on
+    client = init_client(api_key=_key, endpoint=_end, project_id=_pid)
+    users = Users(client)
+    teams = Teams(client)
 
-result = users.list()
-print(result)
+    user_data = load_user_data(args.file)
+    for user in user_data:
```

Comment:
I think, this for-loop would fit well into it's own function `register_user_with_team`

Reply:
For this i would have to pass through almost all variables of the main, I don't think that would that much of an improvement.

Reply:
Okay. It needs `user`, `users`, `teams` and `appwrite_endpoint` but since these variables are "global" you wouldn't need to pass them as parameters to the function.

I only thought about a function name which summarizes what happens. Not mandatory but would be nice.